Dockerfile

```diff
@@ -2,14 +2,20 @@ FROM usgswma/python:3.8
 
 ENV PYTHONUNBUFFERED 1
 
-WORKDIR $HOME/application
-
 COPY . $HOME/application
 
-RUN pip install --no-cache-dir -r requirements.txt
+WORKDIR $HOME/application/wellregistry
+
+RUN pip install --no-cache-dir -r ../requirements.txt
 
 USER $USER
 
 EXPOSE 8000
 
-CMD ["gunicorn", "--chdir", "wellregistry", "--config", "wellregistry/gunicorn.conf.py", "wellregistry.wsgi"]
\ No newline at end of file
+# Run the Django migrations to ensure the DB tier is up to date.
+# Django, like liquibase, executes each entry once.
+# The order below is important initial database configuration.
+CMD python -m manage migrate --database=postgres postgres
+ && python -m manage migrate registry 0000
+ && python -m manage migrate registry
+ && gunicorn --config wellregistry/gunicorn.conf.py wellregistry.wsgi
```

Comment:
This looks fine. Just make sure this doesn't cause container start failures when running in ECS. Otherwise this is good to go.

Reply:
I are referring to it taking too long to start up? I suspect it will be fine but I had planned to keep an eye on that.